Rebuild-Black-Business/RBB-Website
# Volunteer Text from "Submit Information" to "Join us"

# Describe your PR
Submit information -> Join us

![image](https://user-images.githubusercontent.com/8475305/84326071-27b00c80-ab4a-11ea-9cee-ddffd2733ccb.png)

![image](https://user-images.githubusercontent.com/8475305/84326078-2aaafd00-ab4a-11ea-905d-95d68aaac652.png)

Related to #
Fixes #

## Pages/Interfaces that will change
/about volunteer card button
/home volunteer button

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

## Steps to test

1. Go to home
2. scroll down to volunteer section
3. Go to about
4. Scroll down to volunteer card

should read "Join us"

### Additional notes
